### PR TITLE
Added hooks for psutil

### PR DESCRIPTION
**Pull request by byehack**

### CODE REVIEW

File: requirements-test-libraries.txt [hunk 1]
```diff
@@ -135,6 +135,7 @@ xyzservices==2023.7.0
 mistune==3.0.1
 pydantic==2.1.1
 jsonschema==4.19.0
+psutil==5.9.5
```
Comment (bwoodsend) [inline requirements-test-libraries.txt:138]: You need to [supply a test](https://github.com/pyinstaller/pyinstaller-hooks-contrib#test) for this to actually do anything.

File: src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-psutil.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+# A fully customizable messagebox for customtkinter!
+# (extension/add-on)
+# ------------------------------------------------------------------
```
Comment (bwoodsend) [inline src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-psutil.py:14]: This stuff about customtkinter shouldn't be here.

**Review by bwoodsend — CHANGES_REQUESTED** [review]
We'll also need a [news entry](https://github.com/pyinstaller/pyinstaller-hooks-contrib#add-a-news-entry).

File: news/623.new.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+Add a hook for ``psutil``, which has platform independent exclude list.
```
Comment (rokm) [inline news/623.new.rst:1]: The exclude list is platform-dependent, is it not?

Comment (byehack) [reply news/623.new.rst:1]: ah yes, my bad

**Review by bwoodsend — APPROVED** [review]

**Comment by bwoodsend** [comment]
Thanks @byehack

**Pull request merged**